tests/templates.py

```diff
@@ -94,16 +95,16 @@ def render_script(template: str, out_path: str, params: Dict):
     os.chmod(out_path, stat.S_IREAD | stat.S_IWRITE | stat.S_IEXEC)
 
 
-def autogen_cargo(conf_file, yaml: Dict):
-    def render_stage(stage_conf: Mapping[str, Any] | None, filename: str) -> bool:
+def autogen_cargo(conf_file, yaml: Dict) -> Generator[Path]:
+    def render_stage(stage_conf: Mapping[str, Any] | None, filename: str) -> Generator[Path]:
```

Comment:
It would be nice to comment what these return, now that it's less trivial.

Reply:
Fixed in 3dea6a9.